generator.yml

```diff
@@ -55,22 +55,29 @@ defconfigs:
   - &riscv    {config: defconfig,                 << : *riscv-triple}
   - &s390     {config: defconfig,                 << : *s390-triple}
   - &x86_64   {config: defconfig}
+tiers:
+  #                      LLVM=1       LLVM_IAS=1       Make variables to pass to TuxSuite
+  - &llvm_full          {llvm: true,  llvm_ias: true,  make_variables: {LLVM: 1, LLVM_IAS: 1}}
+  - &llvm               {llvm: true,  llvm_ias: false, make_variables: {LLVM: 1}}
+  - &clang              {llvm: false, llvm_ias: false, make_variables: {CC: clang}}
+  - &mips_llvm          {llvm: true,  llvm_ias: false, make_variables: {LD: mips-linux-gnu-ld, LLVM: 1}}
+  - &riscv_llvm_full    {llvm: true,  llvm_ias: true,  make_variables: {LD: riscv64-linux-gnu-ld, LLVM: 1, LLVM_IAS: 1}}
```

Comment:
so these would be:
```sh
$ ARCH=mips CROSS_COMPILE=mips-linux-gnu- make LLVM=1 LD=mips-linux-gnu-ld
$ ARCH=riscv CROSS_COMPILE=riscv64-linux-gnu- make LLVM=1 LLVM_IAS=1 LD=riscv64-linux-gnu-ld
```
?
(Does that work, I need to reread the Makefile's LLVM=1 support, I thought LD couldn't be specified on top of LLVM=1, but I haven't looked and don't remember).

Reply:
Yes, it does. That is how we were doing things in the first CI repo. `LLVM=1` sets the default values but they can still be overridden individually. We could cut most things over to `LLVM=1` and just allowlist the GNU utils that we absolutely need.